frontend/ui/widget/configdialog.lua

```diff
@@ -87,7 +92,7 @@ function OptionTextItem:onTapSelect()
     self.underline_container.color = Blitbuffer.COLOR_BLACK
     self.config:onConfigChoose(self.values, self.name,
                     self.event, self.args,
-                    self.events, self.current_item, self.hide_on_apply)
+                    self.events, self.current_item, self.hide_on_apply, Notification.SOURCE_BOTTOM_MENU_ICON)
```

Comment:
Can't you (here and everywhere below) avoid passing the source, if you Notification:setNotificationSource() it before ?
```lua
Notification:setNotificationSource(Notification.SOURCE_BOTTOM_MENU_ICON)
self.config:onConfigChoose(self.values, self.name, ... as it is)
```